frontend/libs/studio-components/src/components/StudioSpinner/StudioSpinner.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StudioSpinner } from './StudioSpinner';
+import { textMock } from '../../../../../testing/mocks/i18nMock';
+
+const mockSpinnerText: string = 'Test text';
+const mockTestId: string = 'testId';
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useId: () => mockTestId,
+}));
+
+describe('StudioSpinner', () => {
+  afterEach(jest.clearAllMocks);
+
+  it('should render the spinner without spinnerText when it is not present', () => {
```

Comment:
I think it more important to communicate the accessibility testing here, not that the spinnerText is not presented. 😊 
```suggestion
  it('should render default loading message as accessibility title when spinnerText is not provided', () => {
```